### ISSUE TITLE
ssh host keys not generated on boot

### ISSUE THREAD

**Issue by mgerdts** [joyent/linux-live#3]
During boot, the ssh host keys should be generated so that sshd is useful.

Workaround:

```
# dpkg-reconfigure openssh-server
```

You will probably also need to do something like the following to allow root login via ssh.

```
sed -i -e 's/^#PermitRootLogin .*/PermitRootLogin yes/' /etc/ssh/sshd_config
service ssh restart
passwd root
```

**Comment by mgerdts** [comment]
Perhaps the real problem is that the keys are generated as part of the installation.  It would probably be better to use the `Dpkg::ConfigurePending` option from `apt.conf(5)` do prevent `apt` or `dpkg` from configuring it and other software in the first place.  A first boot script should then use [debconf](https://wiki.debian.org/debconf) to ask whatever questions are needed.  Note that answers can be automated with [preseeding](https://www.debian.org/releases/buster/amd64/apbs03.en.html)

**Comment by mgerdts** [comment]
Fixed in 22707898da823a5d6c9656747989e91c151b2fa5
